packages/trader/src/Stores/Modules/Trading/trade-store.ts

```diff
@@ -621,7 +630,12 @@ export default class TradeStore extends BaseStore {
 
         await this.setActiveSymbols();
         await this.root_store.active_symbols.setActiveSymbols();
-        if (should_set_default_symbol) await this.setDefaultSymbol();
+        const { symbol, showModal } = getTradeURLParams({ active_symbols: this.active_symbols });
+        if (showModal && should_show_loading) this.root_store.ui.toggleUrlUnavailableModal(true);
```

Comment:
should_show_loading is true when the app is loading for the first time. I'm using it in order to avoid showing the modal after the app has already loaded but the user is switching between accounts.